### PR TITLE
Render 7 and 30 day average and persist todays time after reset

### PR DESCRIPTION
**Pull request by zwsaile**
# Description

Add 7 day and 30 day averages to the appropriate fields.
Allow the timer data to persist in Today's Time section after timer reset
Style updates to timer and dropdown button

Still needs some final tweaks to format the times properly, I wasn't able to figure it out

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [x] Cosmetic changes (changes to layout or visuals in CSS file)

## How Has This Been Tested?

- [x] console.log()
- [ ] dev tools
- [x] functionality check through web browser
- [ ] tested by viewing the markdown file preview
- [ ] tested with testing framework

## Checklist:

- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have deleted all comments/console.log() after functionality and understanding is final
- [x] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers

### CODE REVIEW

File: src/components/FindUser.js [hunk 1]
```diff
@@ -2,13 +2,15 @@ import { useQuery, gql } from '@apollo/client';
...
 
-function FindUser({ weeklyAverageShowerTime, setWeeklyAverageShowerTime}) {
+function FindUser({ setWeeklyAverageShowerTime, setThirtyDayAverageShowerTime, setUsername}) {
     const { loading, error, data } = useQuery(GET_USER);
 
     useEffect(() => {
         if (data) {
             console.log(data.getUser.weeklyAverageShowerTime)
             setWeeklyAverageShowerTime(data.getUser.weeklyAverageShowerTime)
+            setThirtyDayAverageShowerTime(data.getUser.thirtydayAverageShowerTime)
```
Comment (nbalmoori) [inline src/components/FindUser.js:12]: glad this works now!

File: src/components/ShowerInfo.js [hunk 2]
```diff
@@ -14,34 +24,47 @@ const ShowerInfo = ({ todaysSeconds, todaysMinutes, setTodaysSeconds, setTodaysM
...
-          <h3 className='shower-data-text'>The Week's Total</h3>
+          <h3 className='shower-data-text'>7 Day Average</h3>
+          <h3 className='shower-data-text'>30 Day Average</h3>
         </div>
         <div className='shower-data-num-box'>
-          <h3 className='shower-data-num'>6:19</h3>
-          <h3 className='shower-data-num'>{todaysMinutes}:{todaysSeconds}</h3>
-          <h3 className='shower-data-num'>7:12</h3>
+          <h3 className='shower-data-num'>{totalMinutes}:{totalSeconds}</h3>
+          <h3 className='shower-data-num'>{Math.floor(weeklyAverageShowerTime/60)}:{weeklyAverageShowerTime % 60}</h3>
```
Comment (nbalmoori) [inline src/components/ShowerInfo.js:66]: looks good!

**Review by nbalmoori — COMMENTED** [review]
These changes looked good to me!

File: src/components/ShowerInfo.js [hunk 3]
```diff
@@ -14,34 +24,47 @@ const ShowerInfo = ({ todaysSeconds, todaysMinutes, setTodaysSeconds, setTodaysM
...
+          <h3 className='shower-data-num'>{Math.floor(thirtyDayAverageShowerTime/60)}:{thirtyDayAverageShowerTime % 60}</h3>
```
Comment (rachellhill) [inline src/components/ShowerInfo.js:67]: Nice work on this, Zach!

**Review by rachellhill — APPROVED** [review]
Thanks for doing this Zach!

File: src/components/App.js [hunk 4]
```diff
@@ -3,28 +3,36 @@ import Nav from './Nav'
...
 
   const [todaysSeconds, setTodaysSeconds] = useState(0);
   const [todaysMinutes, setTodaysMinutes] = useState(0);
+  const [totalSeconds, setTotalSeconds] = useState(0);
+  const [totalMinutes, setTotalMinutes] = useState(0);
   const [containerCount, setContainerCount] = useState(0);
   const [bagCount, setBagCount] = useState(0);
   const [strawCount, setStrawCount] = useState(0);
   const [weeklyAverageShowerTime, setWeeklyAverageShowerTime] = useState(0);
+  const [thirtyDayAverageShowerTime, setThirtyDayAverageShowerTime] = useState(0);
```
Comment (whiteheadol) [inline src/components/App.js:20]: Heck yes another query!

File: src/components/App.js [hunk 5]
```diff
@@ -33,6 +41,13 @@ function App() {
           setTodaysSeconds={setTodaysSeconds}
           todaysMinutes={todaysMinutes}
           setTodaysMinutes={setTodaysMinutes}
+          weeklyAverageShowerTime={weeklyAverageShowerTime}
```
Comment (whiteheadol) [inline src/components/App.js:44]: Props for days. I feel like this app could be a possible use case for global state management, but that seems like a lot to try to integrate after this much functionality.

**Pull request merged**